src/python/bezier/_plot_helpers.py

```diff
@@ -62,14 +62,15 @@ def add_plot_boundary(ax, padding=0.125):
     )
 
 
-def add_patch(ax, color, pts_per_edge, *edges):
+def add_patch(ax, color, pts_per_edge, *edges, alpha=None):
```

Comment:
This should just be
```suggestion
def add_patch(ax, color, pts_per_edge, *edges, alpha=0.625):
```
based on the usage below, right?

Reply:
Looking at the rest of the changes, I think it's fine to default to `alpha=0.625` everywhere.

Though I appreciate why you went with `alpha=None` as the API choice.

Reply:
Np, I'll switch over to 0.625 then